library/network-tools/curl/http3/8.11/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine/curl-http3:8.11.0 AS curl-http3
```

Comment:
Why are some using `alpine` and some `debian` base images?

Reply:
Alpine does not include some of the required programs in the stable repository, such as `dig` or `host`. Also, some of the tools such as `ping` and `traceroute` come included inside the `busybox` binary, not as standalone programs, which I consider it to be overkill. Working with programs compiled with the musl libc is easier to work with, which is why I ported with `alpine` as base as much as possible.

Reply:
OK.